Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM registry.fedoraproject.org/fedora-minimal:36
+
+RUN microdnf \
+      --assumeyes \
+      --nodocs \
+      --setopt=install_weak_deps=0 \
+      install \
+        acpica-tools \
+        g++ \
+        gcc \
+        gcc-aarch64-linux-gnu \
+        gcc-arm-linux-gnu \
+        gcc-riscv64-linux-gnu \
+        git \
+        libX11-devel \
+        libXext-devel \
+        libuuid-devel \
+        make \
+        nuget \
+        nasm \
+        python \
+        python3-distutils-extra \
+        python3-pip \
+        python3-setuptools \
+        qemu-system-aarch64-core \
```

Comment:
Do we know how much size the QEMU packages add? I'm inclined to say this is the type of thing we should create a second layer container for. i.e. have fedora-36 & fedora-36-qemu. What do you think?